README.rst

```diff
@@ -101,37 +119,45 @@ Now go to the ``data-model`` directory and run the ``./init_qwat.sh`` script:
+Optionally, you can restore a sample dataset. For that you need to download the data sample dump and restore it into the QWAT database:
+
+::
+
+    QWAT_VERSION=`cat system/CURRENT_VERSION.txt`
+    # With wget
+    wget -q -O qwat_dump.backup "https://github.com/qwat/qwat-data-model/releases/download/$QWAT_VERSION/qwat_v"$QWAT_VERSION"_data_only_sample.backup"
+    pg_restore -U postgres --dbname qwat -e --no-owner --verbose --jobs=3 --disable-triggers --port 5432 qwat_dump.backup
+
+Second way. Create a database from the sample dump using ``pg_restore``
+```````````````````````````````````````````````````````````````````````
+
+In your shell:
 
 ::
 
-        QWAT_VERSION=`cat system/CURRENT_VERSION.txt`
-        # With wget
-        wget "https://github.com/qwat/qwat-data-model/releases/download/$QWAT_VERSION/qwat_v"$QWAT_VERSION"_data_only_sample.backup"
-        pg_restore -U postgres --dbname qwat -e --no-owner --verbose --jobs=3 --disable-triggers --port 5432 "qwat_v"$QWAT_VERSION"_data_only_sample.backup"
-
-        # or directly with curl
-        curl -L "https://github.com/qwat/qwat-data-model/releases/download/$VERSION/qwat_v"$VERSION"_data_only_sample.backup" | pg_restore -U postgres --dbname qwat -e --no-owner --verbose --disable-triggers --port 5432
-
-.. Doesn't work with 1.3.1 see https://github.com/qwat/qwat-data-model/issues/224
-.. Instead of running the init script or git, just run :
-
-.. ::
-  cd ..
-  git clone https://github.com/qwat/qwat-data-sample
-  psql -U postgres -c 'create database qwat;'
-  psql -U postgres -d qwat -c 'create extension postgis;'
-  psql -U postgres -d qwat -c 'create extension hstore;'
-  QWAT_VERSION=1.3.1
-  wget "https://github.com/qwat/qwat-data-model/releases/download/$QWAT_VERSION/qwat_v"$QWAT_VERSION"_data_and_structure_sample.backup"
-  pg_restore -U postgres --dbname qwat -e --no-owner --verbose --jobs=3 --disable-triggers --port 5432 "qwat_v"$QWAT_VERSION"_data_and_structure_sample.backup"
-  # or with curl
-  curl -L "https://github.com/qwat/qwat-data-model/releases/download/$VERSION/qwat_v"$VERSION"_data_and_structure_sample.backup" | pg_restore -U postgres --dbname qwat -e --no-owner --verbose --disable-triggers --port 5432
+    # Create the database and the extensions
+    psql -U postgres -c 'create database qwat;'
+    psql -U postgres -d qwat -c 'create extension postgis;'
+    psql -U postgres -d qwat -c 'create extension hstore;'
+
+    # Create the roles for QWAT
+    psql -c 'CREATE ROLE qwat_viewer NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE NOREPLICATION;' -U postgres
+    psql -c 'CREATE ROLE qwat_user NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE NOREPLICATION;' -U postgres
+    psql -c 'CREATE ROLE qwat_manager NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE NOREPLICATION;' -U postgres
+    psql -c 'CREATE ROLE qwat_sysadmin NOSUPERUSER INHERIT NOCREATEDB NOCREATEROLE NOREPLICATION;' -U postgres
+
+    # Download the latest version
+    QWAT_VERSION=1.3.1
```

Comment:
Instead of hardcoding the QWAT version here I'd just point the user to the latest release page (https://github.com/qwat/qwat-data-model/releases/latest), and tell him or her to download the data_and_structure dump from this page. We could have that at the beginning of  the section, just before providing the command lines to be used to create the database and extensions, etc.

What do you think?

Reply:
This is especially where I would like to have backups with the name "latest":)

It's just a question of habit, on windows, I'll download what I need and do everything in the clients (pgadmin for example). On unix, I open my console and do everything in the shell.

I think I will leave this formulation for the time being, but I will take your proposal in the next major restructuring of doc.

Reply:
> This is especially where I would like to have backups with the name "latest":)

I know :)

> I think I will leave this formulation for the time being, but I will take your proposal in the next major restructuring of doc.

Sure.